src/stack.js

```diff
@@ -0,0 +1,45 @@
+'use strict'
+
+export default class Stack {
+  constructor(initialValues=[]) {
+    this.elements = initialValues
+  }
+
+  push(value) {
+    this.elements[this.elements.length] = value
+  }
+
+  pop() {
```

Comment:
if you're going to use an array internally to implement this you might as well just use pop

```js
pop() {
  if (this.isEmpty()) return null;
  return this.elements.pop()
}
```

doing this by hand would look something like this:
```js
pop() {
  if (this.isEmpty()) return null;
  const member = this.elements[this.size-1]
  delete this.elements[this.size-1]
  this.size--
  return return member
}
```